docs/user-guide/misc/comparison.md

```diff
@@ -0,0 +1,33 @@
+# Comparison with other tools
+
+These are several libraries and tools that share similar functionalities with Polars. This often leads to questions from data experts about what the differences are. Below is a short comparison between some of the more popular data processing tools and Polars, to help data experts make a deliberate decision on which tool to use.
+
+You can find performance benchmarks (h2oai benchmark) of these tools here: [Polars blog post](https://pola.rs/posts/benchmarks/) or a more recent benchmark [done by DuckDB](https://duckdblabs.github.io/db-benchmark/)
+
+### Pandas
+
+Pandas stands as a widely-adopted and comprehensive tool in Python data analysis, renowned for its rich feature set and strong community support. However, due to its single threaded nature, it can struggle with performance and memory usage on medium and large datasets.
+
+In contrast, Polars is optimised for high-performance multithreaded computing on single nodes, providing significant improvements in speed and memory efficiency, particularly for medium to large data operations. Its more composable and stricter API results in greater expressiveness and fewer schema-related bugs.
+
+### Dask
+
+Dask extends Pandas' capabilities to large, distributed datasets. Dask mimics Pandas' API, offering a familiar environment for Pandas users, but with the added benefit of parallel and distributed computing.
+
+While Dask excels in scaling Pandas workflows across clusters, Polars offers superior performance for data processing tasks that can be handled within the constraints of a single node. The choice between Dask and Polars often hinges on the familiarity with the Pandas API and the need for distributed processing for extremely large datasets versus the need for speed and efficiency in a vertical scaled environment.
```

Comment:
> The choice between Dask and Polars often hinges on the familiarity with the Pandas API and the need for distributed processing for extremely large datasets versus the need for speed and efficiency in a vertical scaled environment.

I would say that polars API is far more versatile here as dask only supports a subset of the pandas API.